inc/functions-child.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+// Dequeue parent styles for re-enqueuing in the correct order
+function dequeue_parent_style() {
+	wp_dequeue_style( 'tna-styles' );
+	wp_deregister_style( 'tna-styles' );
+}
+
+// Enqueue styles in correct order
+function tna_child_styles() {
+	wp_register_style( 'tna-parent-styles', get_template_directory_uri() . '/css/base-sass.css.min', array(),
+		EDD_VERSION, 'all' );
+	wp_register_style( 'tna-child-styles', get_stylesheet_directory_uri() . '/css/currency-sass.css.min', array(), '0.1', 'all' );
+	wp_enqueue_style( 'tna-parent-styles' );
+	wp_enqueue_style( 'tna-child-styles' );
+}
+
+function tna_child_scripts() {
```

Comment:
The `tna_child_scripts()` should be re-factored.
A lot of scripts and styles have been enqueued, when it's not needed.

Please refer to the links below.
* https://github.com/nationalarchives/tna-base-child/blob/develop/inc/functions-child.php
* https://github.com/nationalarchives/tna-child-web-archive/blob/master/functions.php

Reply:
Hi @punalsc could you please help me with this as I am not sure what to remove.